pip-requirements.txt

```diff
@@ -1,2 +1,5 @@
 pika==0.9.8
 redis==2.10.1
+urllib3
+certifi
+requests[security]
```

Comment:
overall it is looking good, it will be good to add package version to reduce chance for conflicts with other extensions